packages/lodestar/test/e2e/network/util.ts

```diff
@@ -0,0 +1,15 @@
+import {ENR, Discv5Discovery} from "@chainsafe/discv5";
+import PeerId from "peer-id";
+
+import {NodejsNode} from "../../../src/network/nodejs";
+import {createPeerId, initializePeerInfo} from "../../../src/network";
+import defaults from "../../../src/network/options";
+
+export async function createNode(multiaddr: string, inPeerId?: PeerId): Promise<NodejsNode> {
```

Comment:
How is this new function, we already had this

Reply:
the other `createNode` is in unit test.
The difference here is I want an optional peer id and use only `Discv5Discovery` as discovery method.

Reply:
can you extract it to some common utils so we don't duplicate. Any reason why using only discv5?

Reply:
yeah will do. For this e2e test, I want to make sure nodeA to connect to nodeB thru discv5 after we find by subnet. If we use `MDNS`, it'll connect to each other automatically.